build/config.gypi

```diff
@@ -0,0 +1,67 @@
+# Do not edit. File was generated by node-gyp's "configure" step
```

Comment:
Is this an auto-generated file we should not be git tracking? Would you mind looking this up to see what the best practice is? Thanks!